rbkit-lib/subscriber.cpp

```diff
@@ -55,6 +55,12 @@ void Subscriber::triggerGc() {
     commandSocket->sendCommand(triggerGC_Command);
 }
 
+void Subscriber::startCPUProfiling() {
+    RBKit::CmdStartProfile startCPUProfile;
```

Comment:
added `start_cpu_profiling`
also added `stop_cpu_profiling` in branch #172 

had to add new classes in ```rbcommands.h``` and ```rbcommands.cpp```